### PR TITLE
Reworked Zoom controls. Closes #254

### LINKED ISSUE #254: Allow it to zoom out more of Logisim schematics
Sometimes Logisim schematics get very wide and won't fit on a screen if manually zooming out. The zoom is currently limited to be >= 20 %. The `Auto` button already allows to zoom out below this value.

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
Reworked Zoom controls for wider ranges and better granularity. Also added quick zoom buttons and change the way zoom value is shown.

ss

Closes #254

### CODE REVIEW

File: src/main/resources/resources/logisim/strings/gui/gui_de.properties [hunk 1]
```diff
@@ -109,6 +109,8 @@ attributeDialogTitle = Wert auswählen
 # generic/ZoomControl.java
 #
 zoomShowGrid = Gitternetz umschalten
+# ==> zoomAuto =
+
```
Comment (R3dst0ne) [inline src/main/resources/resources/logisim/strings/gui/gui_de.properties:113]: Please remove the blank line.

File: src/main/resources/resources/logisim/strings/gui/gui_el.properties [hunk 2]
```diff
@@ -109,6 +109,8 @@ attributeDialogTitle = Επιλογή Τιμής
 # generic/ZoomControl.java
 #
 zoomShowGrid = Εναλλαγή εμφάνισης πλέγματος
+# ==> zoomAuto =
+
```
Comment (R3dst0ne) [inline src/main/resources/resources/logisim/strings/gui/gui_el.properties:113]: Please remove the blank line.

File: src/main/resources/resources/logisim/strings/gui/gui_es.properties [hunk 3]
```diff
@@ -109,6 +109,8 @@ attributeDialogTitle = Seleccionar valor
 # generic/ZoomControl.java
 #
 zoomShowGrid = Activar o desactivar cuadrícula
+# ==> zoomAuto =
+
```
Comment (R3dst0ne) [inline src/main/resources/resources/logisim/strings/gui/gui_es.properties:113]: Please remove the blank line.

File: src/main/resources/resources/logisim/strings/gui/gui_it.properties [hunk 4]
```diff
@@ -109,6 +109,8 @@ attributeDialogTitle = Seleziona Valore
 # generic/ZoomControl.java
 #
 zoomShowGrid = Attiva se la griglia è mostrata
+# ==> zoomAuto =
+
```
Comment (R3dst0ne) [inline src/main/resources/resources/logisim/strings/gui/gui_it.properties:113]: Please remove the blank line.

File: src/main/resources/resources/logisim/strings/gui/gui_ja.properties [hunk 5]
```diff
@@ -109,6 +109,8 @@ attributeDialogTitle = 選択値
 # generic/ZoomControl.java
 #
 zoomShowGrid = グリッドを表示するかどうかを切り替える
+# ==> zoomAuto =
+
```
Comment (R3dst0ne) [inline src/main/resources/resources/logisim/strings/gui/gui_ja.properties:113]: Please remove the blank line.

File: src/main/resources/resources/logisim/strings/gui/gui_nl.properties [hunk 6]
```diff
@@ -109,6 +109,8 @@ attributeDialogTitle = Selecteer waarde
 # generic/ZoomControl.java
 #
 zoomShowGrid = Selecteer of grid wordt weergegeven.
+# ==> zoomAuto =
+
```
Comment (R3dst0ne) [inline src/main/resources/resources/logisim/strings/gui/gui_nl.properties:113]: Please remove the blank line.

File: src/main/resources/resources/logisim/strings/gui/gui_pt.properties [hunk 7]
```diff
@@ -109,6 +109,8 @@ attributeDialogTitle = Selecionar valor
 # generic/ZoomControl.java
 #
 zoomShowGrid = Alternar se grade em uso
+# ==> zoomAuto =
+
```
Comment (R3dst0ne) [inline src/main/resources/resources/logisim/strings/gui/gui_pt.properties:113]: Please remove the blank line.

File: src/main/resources/resources/logisim/strings/gui/gui_ru.properties [hunk 8]
```diff
@@ -109,6 +109,8 @@ attributeDialogTitle = Выбор значения
 # generic/ZoomControl.java
 #
 zoomShowGrid = Переключить отображение сетки
+# ==> zoomAuto =
+
```
Comment (R3dst0ne) [inline src/main/resources/resources/logisim/strings/gui/gui_ru.properties:113]: Please remove the blank line.

File: src/main/resources/resources/logisim/strings/gui/gui.properties [hunk 9]
```diff
@@ -723,5 +724,3 @@ testResults = Passed: %s, Failed: %s
 testRunning = Running %s vectors ...
 testSetupFailed = Error preparing test vector: %s
 untitled = Untitled
-
```
Comment (R3dst0ne) [inline src/main/resources/resources/logisim/strings/gui/gui.properties:726]: Two blank lines can be found at the end of each .properties file. For consistency’s sake, do not remove them.

**Review by R3dst0ne — CHANGES_REQUESTED** [review]

File: src/main/resources/resources/logisim/strings/gui/gui_de.properties (hunk 1 above)
Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui_de.properties:113]: Done

File: src/main/resources/resources/logisim/strings/gui/gui_es.properties (hunk 3 above)
Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui_es.properties:113]: Done

File: src/main/resources/resources/logisim/strings/gui/gui_ru.properties (hunk 8 above)
Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui_ru.properties:113]: Done

File: src/main/resources/resources/logisim/strings/gui/gui_pt.properties (hunk 7 above)
Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui_pt.properties:113]: Done

File: src/main/resources/resources/logisim/strings/gui/gui_nl.properties (hunk 6 above)
Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui_nl.properties:113]: Done

File: src/main/resources/resources/logisim/strings/gui/gui.properties (hunk 9 above)
Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui.properties:726]: What are they for?

File: src/main/resources/resources/logisim/strings/gui/gui_ja.properties (hunk 5 above)
Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui_ja.properties:113]: Done

File: src/main/resources/resources/logisim/strings/gui/gui_it.properties (hunk 4 above)
Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui_it.properties:113]: Done

File: src/main/resources/resources/logisim/strings/gui/gui_el.properties (hunk 2 above)
Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui_el.properties:113]: Done

File: src/main/resources/resources/logisim/strings/gui/gui.properties (hunk 9 above)
Comment (R3dst0ne) [reply src/main/resources/resources/logisim/strings/gui/gui.properties:726]: ¯\\_(ツ)_/¯
I have no idea. They were there, so I left them.
I think we can remove them. But should do so in a separate PR, where all of them get removed and the [syncTranslations tool](https://github.com/logisim-evolution/logisim-evolution/blob/develop/tools/syncTranslations.py) gets updated (remove `+ '\n\n'` in line 47 (L.73 in #816)).

Comment (MarcinOrlowski) [reply src/main/resources/resources/logisim/strings/gui/gui.properties:726]: Ok. let's remove this in separate commit that would sync all files

**Pull request merged**